packages/braid-design-system/lib/components/Autosuggest/Autosuggest.docs.tsx

```diff
@@ -597,53 +597,51 @@ const docs: ComponentDocs = {
         <>
           <Text>
             If no suggestions are available and you’d like to provide messaging
-            to the user, you can provide an object with a{' '}
-            <Strong>message</Strong> property to the{' '}
-            <Strong>suggestions</Strong> prop.
+            to the user, you can provide it via the{' '}
+            <Strong>noSuggestionsMessage</Strong> prop.
+          </Text>
+          <Text>
+            The message can either be a single piece of text, or a more
+            structured prompt by providing an object containing{' '}
+            <Strong>text</Strong> and <Strong>description</Strong>.
           </Text>
         </>
       ),
       Example: ({ id, setDefaultState, getState, setState, resetState }) =>
         source(
           <>
-            {setDefaultState('value', { text: '' })}
-
-            <Autosuggest
-              showMobileBackdrop
-              scrollToTopOnMobile
-              label="Label"
-              id={id}
-              value={getState('value')}
-              onChange={setState('value')}
-              onClear={() => resetState('value')}
-              suggestions={(value) => {
-                const filteredSuggestions = filterSuggestions(
-                  [
-                    {
-                      text: 'Apples',
-                      value: 1,
-                    },
-                    {
-                      text: 'Bananas',
-                      value: 2,
-                    },
-                    {
-                      text: 'Broccoli',
-                      value: 3,
-                    },
-                    {
-                      text: 'Carrots',
-                      value: 4,
-                    },
-                  ],
-                  value,
-                );
+            {setDefaultState('field1', { text: '' })}
+            {setDefaultState('field2', { text: '' })}
 
-                return filteredSuggestions.length > 0
-                  ? filteredSuggestions
-                  : { message: 'No results found.' };
-              }}
-            />
+            <Stack space="large">
+              <Autosuggest
+                showMobileBackdrop
+                scrollToTopOnMobile
+                label="Label"
+                description="Focus the field to see a simple message"
+                id={id}
+                value={getState('field1')}
+                onChange={setState('field1')}
+                onClear={() => resetState('field1')}
+                suggestions={[]}
+                noSuggestionsMessage="No results found"
+              />
+              <Autosuggest
+                showMobileBackdrop
+                scrollToTopOnMobile
+                label="Label"
+                description="Focus the field to see more structured prompt"
+                id={id}
```

Comment:
These two components have the same id, so `input`s, `button`s and other HTML elements created by `Autosuggest` will have the same id.